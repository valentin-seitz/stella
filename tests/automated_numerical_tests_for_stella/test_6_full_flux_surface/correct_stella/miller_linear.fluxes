       #time        pflx                    vflx                    qflx
    0.0000E+000    0.0000E+000    0.0000E+000   -3.7661E-025    3.2509E-029    0.0000E+000    0.0000E+000
    1.0000E-002    9.0941E-007    9.0941E-007   -2.3231E-023   -2.2467E-023    1.9786E-006    1.9797E-006
    2.0000E-002    1.2392E-006    1.2392E-006   -8.0603E-022    7.8078E-023    2.7236E-006    2.7199E-006
    3.0000E-002    1.2959E-006    1.2959E-006    3.1083E-021    8.6221E-023    2.8670E-006    2.8479E-006
    4.0000E-002    1.2564E-006    1.2564E-006   -1.3205E-021    9.4717E-023    2.7938E-006    2.7557E-006
    5.0000E-002    1.1385E-006    1.1385E-006    4.9747E-021    9.4936E-023    2.5420E-006    2.4837E-006
    6.0000E-002    9.6200E-007    9.6200E-007    2.3274E-021    9.5139E-023    2.1590E-006    2.0800E-006
    7.0000E-002    7.7182E-007    7.7182E-007    2.4275E-021    9.4851E-024    1.7603E-006    1.6580E-006
    8.0000E-002    6.0285E-007    6.0285E-007    1.2648E-021   -6.8676E-023    1.4199E-006    1.2935E-006
    9.0000E-002    4.6149E-007    4.6149E-007    3.2876E-021   -2.0131E-022    1.1313E-006    9.8617E-007
    1.0000E-001    3.7288E-007    3.7288E-007    1.0360E-021   -1.3650E-022    9.4727E-007    7.8938E-007
