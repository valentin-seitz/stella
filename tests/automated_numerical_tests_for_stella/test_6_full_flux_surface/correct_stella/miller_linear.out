 istep=      0 time=  0.0000E+00  |phi|^2=  0.1566E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      1 time=  0.1000E-01  |phi|^2=  0.1653E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      2 time=  0.2000E-01  |phi|^2=  0.8742E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      3 time=  0.3000E-01  |phi|^2=  0.1258E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      4 time=  0.4000E-01  |phi|^2=  0.1036E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      5 time=  0.5000E-01  |phi|^2=  0.9507E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      6 time=  0.6000E-01  |phi|^2=  0.8784E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      7 time=  0.7000E-01  |phi|^2=  0.1075E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      8 time=  0.8000E-01  |phi|^2=  0.1402E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      9 time=  0.9000E-01  |phi|^2=  0.1289E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=     10 time=  0.1000E+00  |phi|^2=  0.1048E-02 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
